rest-api/dao/physical_measurements_dao.py

```diff
@@ -296,9 +296,10 @@ def _update_participant_summary(self, session, obj):
     participant_summary.physicalMeasurementsCreatedSiteId = obj.createdSiteId
     participant_summary.physicalMeasurementsFinalizedSiteId = obj.finalizedSiteId
     participant_summary.lastModified = clock.CLOCK.now()
-    if 'physicalMeasurementsStatus' in obj:
-      # If this is an update it should have this field. New inserts may not.
-      participant_summary.physicalMeasurementsStatus = obj.physicalMeasurementsStatus
+
+    if obj.status and obj.status == PhysicalMeasurementsStatus.CANCELLED:
```

Comment:
fix a bug, update the participant summary when a physical measurement cancel request comes